db_migrator--1.1.0.sql

```diff
@@ -1609,80 +1545,54 @@ BEGIN
    SET LOCAL client_min_messages = warning;
 
    /* loop through all index columns */
-   old_s := '';
-   old_t := '';
-   old_i := '';
-   FOR loc_s, loc_t, ind_name, uniq, wher, colpos, des, is_expr, expr IN
+   FOR sch, tabname, indname, uniq, wher, expr_array, desc_array, is_expr_array IN
       SELECT schema, table_name, i.index_name, ind.uniqueness, ind.where_clause,
-             i.position, i.descend, i.is_expression, i.column_name
+             array_agg(i.column_name ORDER BY i.position),
+             array_agg(i.descend ORDER BY i.position),
+             array_agg(i.is_expression ORDER BY i.position)
       FROM index_columns i
          JOIN indexes ind
             USING (schema, table_name, index_name)
          JOIN tables t
             USING (schema, table_name)
       WHERE t.migrate
         AND ind.migrate
-      ORDER BY schema, table_name, i.index_name, i.position
+      GROUP BY schema, table_name, i.index_name, ind.uniqueness, ind.where_clause
```

Comment:
I know that the `ORDER BY` is not strictly necessary, but I would prefer to have some fixed ordering, so that the order in which the indexes are created is deterministic.
That might be less confusing for the user, and it might ease debugging.

Reply:
You're right! I amended these changes into new commit 5627fcd